cirq-core/cirq/contrib/qasm_import/_parser.py

```diff
@@ -201,6 +201,12 @@ def __init__(self):
         'id': QasmGateStatement(
             qasm_gate='id', cirq_gate=ops.IdentityGate(1), num_params=0, num_args=1
         ),
+        'p': QasmGateStatement(
+            qasm_gate='p',
+            cirq_gate=(lambda params: QasmUGate(0, 0, params[0] / np.pi)),
```

Comment:
I don't think we need QasmUGate here since we have rz (c.f. rx on line 186 above).